pandas/core/arrays/datetimelike.py

```diff
@@ -7,10 +7,12 @@
 
 from pandas._libs import algos, lib
 from pandas._libs.tslibs import (
+    BaseOffset,
```

Comment:
don't we need to make BaseOffset a cpdef class? its cdef only now

Reply:
No; im not sure cpdef class is a thing.